.github/workflows/build-package.yml.template

```diff
@@ -110,15 +110,16 @@ jobs:
         run: |
           (
             cd trigger
-            git tag $CRATE-$VERSION-$TARGET_ARCH
-            git push origin $CRATE-$VERSION-$TARGET_ARCH
+            git tag $CRATE-$VERSION
+            git push origin $CRATE-$VERSION
           )
 
       - name: Releasing assets
-        uses: softprops/action-gh-release@v1
+        uses: svenstaro/upload-release-action@v2
         with:
-          tag_name: $CRATE-$VERSION-$TARGET_ARCH
-          files: |
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          tag_name: $CRATE-$VERSION
```

Comment:
does this syntax also work for github actions variable substitution? I would expect it to give you a literal `$CRATE-$VERSION`.

Reply:
.... oh yeah: we use sed to do the substituion. Carry on.

Reply:
> does this syntax also work for github actions variable substitution? I would expect it to give you a literal `$CRATE-$VERSION`.

Nope, but this syntax is used here because it is a template.

This file already use this syntax before this PR.